conf/types.json

```diff
@@ -0,0 +1,69 @@
+{
+  "carbonblack":{
+    "userName": ["username"],
+    "destinationDomain": ["domain"],
+    "processPath": ["parent_path", "process_path", "path"],
+    "filePath": ["path"],
+    "transportProtocol": ["protocol"],
+    "processName": ["parent_name", "process_name"],
+    "fileName": ["observed_filename", "file_path"],
+    "command": ["cmdline"],
+    "fileHash": ["process_md5", "parent_md5", "expect_followon_w_md5", "md5"],
+    "deviceAddress": ["interface_ip", "comms_ip"],
+    "sourceAddress": ["ipv4", "local_ip"],
+    "destinationAddress": ["remote_ip"],
+    "sourcePort": ["port", "local_port"],
+    "destinationPort": ["remote_port"]
+  },
+  "cloudwatch":{
+    "userName": ["userName", "owner", "invokedBy"],
+    "sourceAccount": ["account"],
+    "destinationAccount": ["recipientAccountId"],
+    "transportProtocol": ["protocol"],
+    "eventType": ["eventType"],
+    "eventName": ["eventName"],
+    "region": ["region"],
+    "userAgent": ["userAgent"],
+    "sourceAddress": ["source", "sourceIPAddress"],
+    "destinationAddress": ["destination"],
+    "sourcePort": ["srcport"],
+    "destinationPort": ["destport"]
+  },
+  "cloudtrail": {
+    "sourceAccount": ["account", "accountId"],
+    "destinationAccount": ["recipientAccountId"],
+    "eventType": ["eventType"],
+    "eventName": ["eventName"],
+    "region": ["region", "awsRegion"],
+    "userAgent": ["userAgent"],
+    "sourceAddress": ["sourceIPAddress"]
+  },
+  "ghe": {
+    "processName": ["program"],
+    "userName": ["current_user"],
+    "destinationAddress": ["remote_address"],
```

Comment:
I don't see an example GHE log with `remote_address` - lets chat offline

Reply:
It is defined in the [GHE schema](https://github.com/airbnb/streamalert/blob/master/conf/logs.json#L785).